README.md

```diff
@@ -16,46 +15,79 @@ Or create a new stock_portfolio and manually add your stocks to it (holdings and
+Counter({'TQQQ': 6, 'LAND': 6, 'XLE': 1})
+```
+The internal dataframe has been updated to reflect the new holdings in the portfolio.
 
-| ticker   | goal_percentage   | holding_percentage   |   holdings |   stock_price |   holdings_value |   goal_holding_value |   holding_value_diff |   need_to_buy_or_sell |
-|:---------|:------------------|:---------------------|-----------:|--------------:|-----------------:|---------------------:|---------------------:|----------------------:|
-| VTI      | 40.0%             | 37.9%                |         21 |        227.92 |          4786.32 |              5047.2  |               260.88 |                1.1446 |
-| VGT      | 10.0%             | 13.1%                |          4 |        414.83 |          1659.32 |              1261.8  |              -397.52 |               -0.9583 |
-| VEA      | 10.0%             | 9.42%                |         23 |         51.69 |          1188.87 |              1261.8  |                72.93 |                1.4109 |
-| VWO      | 10.0%             | 9.29%                |         23 |         50.97 |          1172.31 |              1261.8  |                89.49 |                1.7557 |
-| BND      | 5.0%              | 4.11%                |          6 |         86.48 |           518.88 |               630.9  |               112.02 |                1.2953 |
-| MUB      | 10.0%             | 7.47%                |          8 |        117.83 |           942.64 |              1261.8  |               319.16 |                2.7086 |
-| LQD      | 2.5%              | 2.14%                |          2 |        135.47 |           270.94 |               315.45 |                44.51 |                0.3286 |
-| BNDX     | 2.5%              | 2.28%                |          5 |         57.76 |           288.8  |               315.45 |                26.65 |                0.4614 |
-| XLE      | 5.0%              | 14.1%                |         36 |         49.72 |          1789.92 |               630.9  |             -1159.02 |              -23.3109 |
-| GLD      | 5.0%              | 0.0%                 |          0 |        168.16 |             0    |               630.9  |               630.9  |                3.7518 |  
+```
+chadfolio.dataframe[['ticker', 'holdings']]
+```
+| ticker   |   holdings |
+|:---------|-----------:|
+| TQQQ     |         56 |
+| LAND     |        106 |
+| XLE      |         51 |
 
+
+### stonkers_demo.py
+Below is the command output from running `stonkers.stonkers_demo.py`. Note the final column `need_to_buy_or_sell`, which tells the user how much he needs to buy or sell of each stock so that the holding_percentages align with the goal_percentages.
+```
+Current total portfolio value: 13065.27
+Necessary Portfolio value to rebalance with only buying: 16488.4
+Required Cash Money: 3423.13
+| ticker   | goal_percentage   | holding_percentage   |   holdings |   stock_price |   holding_value_diff |   need_to_buy_or_sell |   need_to_buy |
```

Comment:
What's the difference between the `need_to_buy_or_sell` and `need_to_buy` columns here? Seems confusing looking at the example numbers below

Reply:
need_to_buy_or_sell is if you are balancing your portfolio by buying and selling your current stocks. Hence it allows negative values (selling). It does not require additional cash.

need_to_buy is if you were to balance your portfolio by only buying new stocks. So the most overbought stock will have a zero and everything else will have a positive value indicating how many of the stocks to buy in order to reach the goal percentages.

Both of these calculations are kinda irrelevant now with the `stock_buy_advisor` method which takes a fixed amount of cash and suggests which are the most important stocks to buy and how many in order to reach a more balanced portfolio.